pkgs/development/python-modules/python-chess/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "python-chess";
+  version = "1.6.1";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "chess";
+    sha256 = "sha256-lmumSDVZ3A+fOr1vryvKLhBVBGavQjvnmodlY509zTM=";
+  };
+
+  doCheck = false;
```

Comment:
It looks like that [`test.py`](https://github.com/niklasf/python-chess/blob/master/test.py) could contain unit test. It that file is not part of the source that is shipped through PyPI then we could switch to fetching from GitHub (`fetchFromGitHub`).

Please add `pythonImportsCheck` as additional safe-guard.